### ISSUE TITLE
#offsets not understood

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#74]
Rowan sha: ef4afb
Jadeite: 3.0.14

- Create new method 

```
RowanSample1>>streamArray
	| array ws |
	array := Array with: 'abc' with: 'def'. 
	ws := WriteStream on: String new. 
	1 to: array size do:[:idx |
			ws nextPutAll: (array at: idx).
			ws space.
	].
	^ws contents

```
- Set a breakpoint on the ``new`` in ``String new``
- Turn on the new debugger ``RowanDebugger debuggerClass: RowanDebugger``
- Execute the method in a workspace
- Get dialog ``Error while processing error: RowanFrameService does not understand #offsets``

![image](https://user-images.githubusercontent.com/36212516/44355164-b2e57d00-a460-11e8-9130-341b94e64368.png)

**Comment by jgfoster** [comment]
Fixed with 2c90f868ce1bb15136e548c14b3d7577ed076166.

**Comment by ericwinger** [comment]
After integration of pull request #86 this appears to be fixed.
